modules/install-and-configure-the-splunk-add-on.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-splunk.adoc
+:_module-type: PROCEDURE
+[id="install-and-configure-the-splunk-add-on_{context}"]
+= Installing and configuring the Splunk add-on
+
+You can install the StackRox Kubernetes Security Platform add-on from your Splunk instance.
+
+.Prerequisites
+* You must have an API token with `read` permission for all resources of {product-title}. You can assign the *Analyst* system role to grant this level of access. The *Analyst* role has read permissions for all resources.
+
+.Procedure
+. Download the StackRox Kubernetes Security Platform add-on from link:https://splunkbase.splunk.com/app/5315/[Splunkbase].
+. Navigate to the Splunk home page on your Splunk instance.
+. Navigate to *Apps* > *Manage Apps*.
+. Select *Install app from file*.
+. In the *Upload app* pop-up, select *Choose File* and select the {product-title} add-on file.
```

Comment:
* Should be pop-up <something>
* This uses the product-title to be RHACS add-on file, but the rest used StackRox still. Just checking that that is all correct